README.md

```diff
@@ -1,6 +1,6 @@
-# Jbrowse Prototype
+# DMS Jbrowse Components
 
-This is a NextJS application containing Jbrowse prototypes.
+This package contains a DMS component that makes requests to Song and Score, then displays a Jbrowse visualization.
```

Comment:
```suggestion
This package provides React components that use [JBrowse](https://jbrowse.org/jb2/) to visualize VCF and BAM files sourced from [Song](https://github.com/overture-stack/song) and [Score](https://github.com/overture-stack/score) data repositories. It has been built for use in the [DMS UI](https://github.com/overture-stack/dms-ui) as an optional visualization tool.
```

Reply:
Would also be worth documenting which versions of the song and score services this was built for. A section for dependencies where we list target versions of node, song, score,  might be nice.

Consider also adding a section on how to run this standalone and provide it test data.